test/job/test.sh

```
@@ -23,41 +27,46 @@ export service_account_name=${SERVICE_ACCOUNT_NAME:-"default"}
 server_kubeconfig=${SERVER_KUBECONFIG:-"${PWD}/test/e2e/.kubeconfig"}
 agent_kubeconfig=${AGENT_KUBECONFIG:-"${PWD}/test/e2e/.kubeconfig"}
 
-# these kubeconfigs are used in e2e test job to connect the server and agent clusters
+# these kubeconfigs are used in test job to connect the server and agent clusters
 server_in_cluster_kubeconfig=${SERVER_IN_CLUSTER_KUBECONFIG:-"${PWD}/test/e2e/.in-cluster.kubeconfig"}
 agent_in_cluster_kubeconfig=${AGENT_IN_CLUSTER_KUBECONFIG:-"${PWD}/test/e2e/.in-cluster.kubeconfig"}
 
 timeout=${TIMEOUT:-"30m"}
```

Shellcheck (0.11.0)

[warning] timeout appears unused. Verify use (or export if used externally).

(SC2034)